deploy/rp-development-predeploy.json

```diff
@@ -17,6 +17,28 @@
         }
     },
     "resources": [
+        {
```

Comment:
@asalkeld where is this role used?

Reply:
It is the role which enables the first party service principal to manage ACR tokens.

Reply:
It'll be a one-off task to add it to the global subscription containing the ACR and apply the roll on the ACR itself (once in int, once in prod).